### ISSUE TITLE
[Error Report]: Mod manager crashes immediately on launch

### ISSUE THREAD

**Issue by sonickid1816** [X-Hax/SA-Mod-Manager#181]
This only happens with SA2 installed specifically. The manager works when only SA1 is installed.
This crash happens when I either launch straight into the SA2 mods page or when I switch to it from the SA1 section.
I have already troubleshooted by uninstalling and freshly installing both games and the manager. I even deleted the manager directories in AppData before reinstalling everything. I also ran the manager as admin just in case, but that didn't change anything.
## Exception Report
```
SAMM Information:
	Manager Version: 1.2.10
	Current Game: Sonic Adventure 2
	Game Directory: D:\SteamLibrary\steamapps\common\Sonic Adventure 2
	Manager Location: D:\SteamLibrary\steamapps\common\Sonic Adventure DX\
	Running as Admin: False
```

```
Exception Information:
	Type: InvalidOperationException
	Exception: Property HRTFSound not found or read-only.
	Source: SAModManager
	In Function: Void SavePatches(System.Object ByRef)
	StackTrace:    at SAModManager.Controls.SA2.GameConfig.SavePatches(Object& input)
	   at SAModManager.MainWindow.SaveSA2Settings()
	   at SAModManager.MainWindow.Save()
	   at SAModManager.MainWindow.comboProfile_SelectionChanged(Object sender, SelectionChangedEventArgs e)
	   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
	   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
	   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
	   at System.Windows.Controls.ComboBox.OnSelectionChanged(SelectionChangedEventArgs e)
	   at System.Windows.Controls.Primitives.Selector.SelectionChanger.SelectJustThisItem(ItemInfo info, Boolean assumeInItemsCollection)
	   at System.Windows.DependencyObject.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
	   at System.Windows.FrameworkElement.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
	   at System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)
	   at System.Windows.DependencyObject.UpdateEffectiveValue(EntryIndex entryIndex, DependencyProperty dp, PropertyMetadata metadata, EffectiveValueEntry oldEntry, EffectiveValueEntry& newEntry, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType)
	   at System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
	   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)
	   at System.Windows.Data.BindingOperations.SetBinding(DependencyObject target, DependencyProperty dp, BindingBase binding)
	   at SAModManager.MainWindow.SetManagerBindings()
	   at SAModManager.MainWindow.SetBindings()
	   at SAModManager.MainWindow.MainWindowManager_Loaded(Object sender, RoutedEventArgs e)
	   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
	   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
	   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
	   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
	   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
	   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
	--- End of stack trace from previous location ---
	   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
	   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
	   at System.Windows.Threading.DispatcherOperation.Invoke()
	   at System.Windows.Threading.Dispatcher.ProcessQueue()
	   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
	   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
	   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
	   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
	   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
	   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
	   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
	   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
	   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
	   at System.Windows.Application.RunDispatcher(Object ignore)
	   at System.Windows.Application.RunInternal(Window window)
	   at SAModManager.App.Main()
```

**Comment by sonickid1816** [comment]
Nevermind. It turns out that I just needed to reinstall all of my mods.

**Comment by AuraTheCat** [comment]
I've been having a similar problem. Every time I load in a mod for sa2, it the crashes after the sega and sonic team logo. I've already tried redownloading everything, deleting the samanager folder, everything I tried wasn't working. I got this when I already had sa2 mod manager and just deleted it when i got it. Idk what I did wrong because no files are missing and all the mods are updated. If someone can give ke the answer, pls help me out
